### PR TITLE
Add error logging

### PR DESCRIPTION
**Pull request by buyfn**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,22 +1,51 @@
...
 const app = express();
 
 app.use(express.json());
 
+app.use((req, res, next) => {
+    logger.info({
+        path: req.path,
+        method: req.method,
+        body: req.body,
+        'query params': req.query
```
Comment (titivuk) [inline src/index.js]:
```suggestion
        'queryParams': req.query
```

Comment (titivuk) [reply src/index.js]: or `query`

File: src/logger.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import winston from 'winston';
+
+export const logger = winston.createLogger({
+    transports: [
```
Comment (titivuk) [inline src/logger.js:8]: you can also modify your logging format by providing some essential data like timestamp
```
format: winston.format.combine(
    winston.format.timestamp(),
    winston.format.json()
)
```

**Comment by titivuk** [comment]
also I noticed that group functionality is in user service, You should separate it from users

File: src/index.js (hunk 1 above)
Comment (buyfn) [reply src/index.js]: Changed

File: src/logger.js (hunk 2 above)
Comment (buyfn) [reply src/logger.js:8]: Added this, thanks for sharing!

**Comment by buyfn** [comment]
> also I noticed that group functionality is in user service, You should separate it from users

Separated groups service

**Review by titivuk — APPROVED** [review]

**Pull request merged**